PLUGINS.md

```diff
@@ -2,3 +2,4 @@ This is the list of all known third-party plugins for RocksDB. If something is m
 
 * [Dedupfs](https://github.com/ajkr/dedupfs): an example for plugin developers to reference
 * [ZenFS](https://github.com/westerndigitalcorporation/zenfs): a file system for zoned block devices
+* [RADOS](https://github.com/riversand963/rocksdb-hdfs-env): an Env used for interacting with RADOS. Migrated from RocksDB main repo.
```

Comment:
Can you double check the URL? The RADOS item has "rocksdb-hdfs-env" in it.

Reply:
Fixed.